prog/mutation.go

```diff
@@ -501,7 +501,11 @@ func (t *IntType) getMutationPrio(target *Target, arg Arg, ignoreSpecial bool) (
 		return plainPrio, false
 	}
 
-	switch size := t.RangeEnd - t.RangeBegin + 1; {
+	size := t.RangeEnd - t.RangeBegin + 1
+	if t.Align != 0 {
```

Comment:
We may also need changes to mutateInt because it tries to add/subtract small values every other time.

Reply:
Done in the last update.